docs/catalog.md

```diff
@@ -610,6 +610,9 @@ The following is an example of creating a package binding.
 The `/whisk.system/pushnotifications/sendMessage` action sends push notifications to registered devices. The parameters are as follows:
 - `text`: The notification message to be shown to the user. For example: `-p text "Hi ,OpenWhisk send a notification"`.
 - `url`: An optional URL that can be sent along with the alert. For example: `-p url "https:\\www.w3.ibm.com"`.
+- `deviceIds` The list of specified devices. For example: `-p deviceIds ["deviceID1"]`.
```

Comment:
do you need quotes around the `[]` (and if so then need to escape the inner quotes)?